solr/core/src/test-files/solr/question-answer-repository/repository.json

```diff
@@ -0,0 +1,57 @@
+[
+  {
+    "name": "question-answer",
+    "description": "A natural language question answering plugin",
+    "versions": [
+      {
+        "version": "1.0.0",
+        "date": "2019-01-01",
+        "artifacts": [
+          {
+            "url": "question-answer-request-handler-1.0.jar",
+            "sig": "C9UWKkucmY3UNzqn0VLneVMe9kCbJjw7Urc76vGenoRwp32xvNn5ZIGZ7G34xZP7cVjqn/ltDlLWBZ/C3eAtuw=="
+          }
+        ],
+        "manifest": {
+          "min-solr-version": "8.0",
+          "max-solr-version": "9.99",
```

Comment:
9.99 isn't very compelling example. If we switch to semVer syntax we don't need these hacks, The min-max combination you try to express here could instead be

    "require-solr-version": "8 - 9"